Gruntfile.js

```diff
@@ -20,7 +56,13 @@ module.exports = function(grunt) {
         }
       }
     }
+
+
   });
+
+  grunt.loadNpmTasks('grunt-contrib-copy');
+  grunt.loadNpmTasks('grunt-css-url-replace');
   grunt.loadNpmTasks('grunt-a11y');
-  grunt.registerTask('default', ['a11y']);
+
+  grunt.registerTask('default', ['a11y', 'copy', 'css_url_replace']);
```

Comment:
I need to separate the grunt tasks that build the site from the grunt tasks that test the built site.